### PR TITLE
unable to create pods after setting resources

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1705322

### CODE REVIEW

**Comment by mburke5678** [comment]
@sunilcio PTAL. Thanks!!

**Comment by sunilcio** [comment]
@mburke5678 thanks, lgtm

**Comment by mburke5678** [comment]
@umohnani8 Which OpenShift versions does this change apply to? It looks like the code change went into 4.6. Was it back ported to 4.5 or 4.4? Thank you!

File: modules/deployments-setting-resources.adoc [hunk 1]
```diff
@@ -12,6 +12,11 @@ This resource is available only if a cluster administrator has enabled the ephem
 
 A deployment is completed by a pod that consumes resources (memory, CPU, and ephemeral storage) on a node. By default, pods consume unbounded node resources. However, if a project specifies default container limits, then pods consume resources up to those limits.
 
+[NOTE]
+====
+The minimum memory limit for a depoyment is 12 MB. If a container fails to start due to a `Cannot allocate memory` pod event, the memory limit is too low. You should either increase or remove the memory limit. Removing the limit allows pods to consume unbounded node resources.
```
Comment (kalexand-rh) [inline modules/deployments-setting-resources.adoc:17]:
```suggestion
The minimum memory limit for a deployment is 12 MB. If a container fails to start due to a `Cannot allocate memory` pod event, the memory limit is too low. Either increase or remove the memory limit. Removing the limit allows pods to consume unbounded node resources.
```

File: modules/nodes-cluster-resource-configure-about.adoc [hunk 2]
```diff
@@ -44,6 +44,9 @@ Note the following about memory requests and memory limits:
 
   - The cluster administrator can assign quota or assign default values for the memory limit value.
 
+  - The minimum memory limit is 12 MB. If a container fails to start due to a `Cannot allocate memory` pod event, the memory limit is too low. 
+You should either increase or remove the memory limit. Removing the limit allows pods to consume unbounded node resources.
```
Comment (kalexand-rh) [inline modules/nodes-cluster-resource-configure-about.adoc:48]:
```suggestion
Either increase or remove the memory limit. Removing the limit allows pods to consume unbounded node resources.
```

**Review by kalexand-rh — COMMENTED** [review]
This is looking good!

**Comment by umohnani8** [comment]
@mburke5678 this change should be available in 4.4 and 4.5 as well. The cri-o PR made it in before cri-o 1.15 was cut.

**Pull request merged**